<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_serverIp">
          <property name="text">
           <string>Server ip:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_serverIp">
          <property name="placeholderText">
           <string>&lt;server IP&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_serverPort">
          <property name="text">
           <string>Server port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_serverPort">
          <property name="placeholderText">
           <string>&lt;server Port&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_pseudo">
          <property name="text">
           <string>Pseudo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_pseudo">
          <property name="placeholderText">
           <string>&lt;your pseudo&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_connectDisconnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lw_recvChat"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_clearRecvLw">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lw_trmtChat"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_clearSentLw">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_msgChat">
        <property name="placeholderText">
         <string>&lt;your message&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
